src/preprocessing/passes/bool_to_bv.cpp

```diff
@@ -39,165 +39,159 @@ PreprocessingPassResult BoolToBV::applyInternal(
       options::preprocessStep());
 
   unsigned size = assertionsToPreprocess->size();
-  for (unsigned i = 0; i < size; ++i)
+
+  if (options::boolToBitvector() == BOOL_TO_BV_ALL)
```

Comment:
Perhaps use a switch/case?

Reply:
It's only two cases, so I'd say no to switch/case. But I'd suggest to store option values that are accessed multiple times in a variable so that you have to access the option only once (not performance critical here but a good habit in general to reduce overhead).